inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortServiceImpl.java

```diff
@@ -80,7 +81,15 @@ public List<SortStatusInfo> listSortStatus(SortStatusRequest request) {
 
         try {
             List<InlongGroupInfo> groupInfoList = request.getInlongGroupIds().stream()
-                    .map(groupId -> groupService.get(groupId))
+                    .map(groupId -> {
+                        try {
+                            return groupService.get(groupId);
+                        } catch (Exception e) {
+                            log.debug("{} may have already been deleted, skip it", groupId);
```

Comment:
Why not print the exception

Reply:
The exception has been printed in `groupService.get()`.